Jenkinsfile

```diff
@@ -2,7 +2,7 @@
 
 @Library("sec_ci_libs@v2-latest") _
 
-def master_branches = ["master", ] as String[]
```

Comment:
We don't want to add `feature/DCOS-39712-masters-tab` to master branches on master, do we?

Reply:
we can remove it on this branch, but then we have to approve every single CI run until it got merged…
The idea was to open a cleanup PR directly after this has landed 🤔

Reply:
Why will we need approving PRs? Following Prs will go against master, not this branch as I understand.

Reply:
not PR, Jenkins CI run (security library…)

Reply:
As I understand we have it here because otherwise every other PR against this `feature/DCOS-39712-masters-tab` branch will be considered unsecure. But now we are about to merge the feature branch itself, so there will be no PRs against the feature branch, correct?

Reply:
no, its here because every run of `continuous-integration/jenkins/branch` would be considered insecure, PRs arent affected by this line.

Reply:
Ok and we want to run branch CI? How every other normal PR works then?

Reply:
check this (giant) if block here: https://github.com/mesosphere/sec-ci-libs/blob/master/vars/user_is_authorized.groovy#L77

depending on the given env vars the script checks for different things, as stated in your slack-update-pr, `CHANGE_BRANCH` and `CHANGE_AUTHOR` are set for PRs, so the lib is able to check the author. on a "branch run" both are null which means that the lib has no chance to check anything - so it asks us for permission… this list acts a whitelist (and while typing this, i realize that it is a bad idea to have a non-protected branch in this whitelist :/ -> will try to figure out a solution ASAP)

Reply:
@orlandohohmeier ^ fyi :/

Reply:
Ok, so we need this to be whitelisted because we want to run `continuous-integration/jenkins/branch` for feature branches, correct? My point is as we are going to merge the feature branch itself into master, wouldn't it be enough just having the `continuous-integration/jenkins/pr-merge` to pass?

Reply:
hehe, fair point 😅once we're going to "use" the PR itself, there is no need to the branch CI to run… only thing it that jenkins will annoy us in our channel a few times 🤔

Reply:
It's just that I wanted to ✅ and merge the branch if we update the whitelist. It's good to go without non-leading masters in my opinion.